flink-core/src/main/java/org/apache/flink/core/fs/Path.java

```diff
@@ -146,24 +146,22 @@ public Path(Path parent, Path child) {
 		}
 
 		final URI resolved = parentUri.resolve(child.uri);
-		initialize(resolved.getScheme(), resolved.getAuthority(), normalizePath(resolved.getPath()));
```

Comment:
Why remove `normalizePath`?

Reply:
Because `initialize` will call `normalizePath `, so it's redundant.